build.sbt

```diff
@@ -192,14 +192,10 @@ lazy val chipyard = (project in file("generators/chipyard"))
 
 lazy val tracegen = (project in file("generators/tracegen"))
   .sourceDependency(testchipip, testchipipLib)
-  .dependsOn(rocketchip, sifive_cache, boom, utilities)
+  .dependsOn(rocketchip, sifive_cache, boom)
   .settings(libraryDependencies ++= rocketLibDeps.value)
   .settings(commonSettings)
 
-lazy val utilities = (project in file("generators/utilities"))
-  .sourceDependency(testchipip, testchipipLib)
```

Comment:
Nice. Everything I see this I have to remind myself why we need this. Glad to see it getting removed